sokutei.h

```diff
@@ -0,0 +1,188 @@
+#ifndef SOKUTEI_BENCHMARK_H
+#define SOKUTEI_BENCHMARK_H
+
+#define SOKUTEI_ITERATIONS 10
+#define SOKUTEI_MAX_COUNTER_COUNT  100
+#define SOKUTEI_COUNTER_LENGTH 30
```

Comment:
The name of this macro does not reflect its actual use. The macro sets the **maximum** length of counter **names**. What do you think, wouldn't it be better to include the highlighted words in the macro name too?